master/internal/agent/agent.go

```diff
@@ -69,6 +72,21 @@ func (a *agent) Receive(ctx *actor.Context) error {
 		a.containers[msg.Container.ID] = msg.Task
 	case aproto.MasterMessage:
 		a.handleIncomingWSMessage(ctx, msg)
+	case *proto.GetAgentRequest:
+		ctx.Respond(&proto.GetAgentResponse{Agent: toProtoAgent(a.summarize(ctx))})
+	case *proto.GetSlotsRequest:
+		var slots []*agentv1.Slot
+		for _, s := range a.summarize(ctx).Slots {
+			slots = append(slots, toProtoSlot(s))
+		}
+		sort.Slice(slots, func(i, j int) bool { return slots[i].Id < slots[j].Id })
```

Comment:
non-blocking: not sure what we get out of sorting by ID

Reply:
The idea here is that it sets a standard for ensuring output stays consistent between calls at the API level. This makes rendering tables in the WebUI and CLI more consistent.